parlai/scripts/eval_wordstat.py

```diff
@@ -112,6 +112,7 @@ def eval_wordstat(opt, print_parser=None):
     :param print_parser: if provided, prints the options that are set within the
         model after loading the model
     """
+    print(opt)
```

Comment:
did you mean to add this here?

Reply:
hmm.. sort of did, sort of didn't.. haha..i'll remove